vcsclient/github.go

```diff
@@ -503,6 +503,27 @@ func (client *GitHubClient) UploadCodeScanning(ctx context.Context, owner, repos
 	return "", nil
 }
 
+// DownloadFileFromRepo on GitHub
+func (client *GitHubClient) DownloadFileFromRepo(ctx context.Context, owner, repository, branch, path string) ([]byte, int, error) {
+	ghClient, err := client.buildGithubClient(ctx)
+	if err != nil {
+		return nil, 0, err
+	}
+	body, response, err := ghClient.Repositories.DownloadContents(ctx, owner, repository, path, &github.RepositoryContentGetOptions{Ref: branch})
+	if err != nil {
+		return nil, response.StatusCode, err
+	}
+	if response.StatusCode != http.StatusOK {
+		return nil, response.StatusCode, fmt.Errorf("expected %d status code while received %d status code", http.StatusOK, response.StatusCode)
+	}
+
```

Comment:
Should we close the body here? 
https://github.com/google/go-github/blob/v49.0.0/github/repos_contents.go#L122

Reply:
done